gst.so/src/rvs_module.cpp

```diff
@@ -51,51 +51,51 @@ extern "C" int rvs_module_has_interface(int iid) {
 }
 
 extern "C" const char* rvs_module_get_description(void) {
-    return "ROCm Validation Suite GST module";
+  return "The GPU Stress Test runs a Graphics Stress test or SGEMM/DGEMM/HGEMM (Single/Double/Half-precision General Matrix Multiplication) \n\tworkload on one, some or all GPUs.";
```

Comment:
I dont think "graphics stress test" is right. sufficient to say sgemm etc right?

Reply:
Thats true. The description was taken from FEATURES.md. Will be correcting the description in both the places.